.jshintrc

```diff
@@ -1,6 +1,17 @@
 {
-	"-W058": true,
+	"eqeqeq": true,
+	"immed": true,
+	"newcap": true,
+	"noarg": true,
+	"nonbsp": true,
+	"undef": true,
+	"unused": "vars",
 	"boss": true,
+	"esnext": true,
+	"expr": true,
+	"sub": true,
+	"node": true,
 	"strict": false,
-	"predef": ["Window", "HTMLDocument", "HTMLElement"]
+	"globalstrict": true,
+	"curly": false
```

Comment:
This doesn't seem to be in the spec: http://origami.ft.com/docs/syntax/js/#syntax-convention-rules

Disclaimer: I'm anti-single line branching statements.